src/main/java/uk/gov/justice/digital/delius/controller/advice/SecureControllerAdvice.java

```diff
@@ -109,4 +110,16 @@
                         .build());
     }
 
+    @ExceptionHandler(ResponseStatusException.class)
+    public ResponseEntity<ErrorResponse> handleException(final ResponseStatusException e) {
```

Comment:
Spring Boot upgrade changed the default behaviour for this exception, the message was no longer being propagated in the JSON.
So this is now consistent with other exception handlers